services/src/atdd-staging/src/test/java/org/openkilda/atdd/staging/service/StubServiceFactory.java

```diff
@@ -112,7 +113,7 @@ public FloodlightService getFloodlightStub() {
 
         when(serviceMock.getFlows(any()))
                 .thenAnswer(invocation -> {
-                    String switchId = (String) invocation.getArguments()[0];
+                    SwitchId switchId = new SwitchId(invocation.getArguments()[0].toString());
```

Comment:
`invocation.getArguments()[0]` is SwitchId. Why for you add .toString() => new(String) loop?